src/components/specimen/Specimen.tsx

```diff
@@ -122,11 +122,11 @@ const Specimen = () => {
         const copySpecimenAnnotations = { ...specimenAnnotations };
 
         /* Check if array for target property exists */
-        if (annotation.target.indvProp in specimenAnnotations) {
+        if (annotation['oa:target']['oa:selector']?.['ods:field'] as string in specimenAnnotations) {
```

Comment:
Why is this necessary?

Reply:
Well this is a fun one. It gets complicated because I want to access a one of optional property inside of an already optional property. TypeScript gets really nervous when I do not include the optional (?) and the type definition of the optional one of property. Especially the one of is triggering TypeScript as it can be a string, but also unknown (not there at all). That's why I need to declare it to be a string, assuming it to be there and interpreting it as a string.

Reply:
Ok thanks, sounds like fun